libindy/include/indy_crypto.h

```diff
@@ -317,6 +317,123 @@ extern "C" {
                                                                       indy_u32_t        decrypted_msg_len)
                                                  );
 
+
+    /// Packs a message
+    ///
+    /// Note to use DID keys with this function you can call indy_key_for_did to get key id (verkey)
+    /// for specific DID.
+    ///
+    /// #Params
+    /// command_handle: command handle to map callback to user context.
+    /// message: a pointer to the first byte of the message to be packed
+    /// message_len: the length of the message
+    /// receivers: a string in the format of a json list which will contain the list of receiver's keys
+    ///                the message is being encrypted for.
+    ///                Example:
+    ///                "[<receiver edge_agent_1 verkey>, <receiver edge_agent_2 verkey>]"
+    /// sender: the sender's verkey as a string When "" is used in this parameter, anoncrypt is used
```

Comment:
I suggest to use `null` instead of empty string to switch between modes.

Reply:
There is another option to split into 2 separate methods but I prefer current approach (determine type by parameter value) instead of old one (`anon/auth_crypt`)

@vimmerru any thoughts?

Reply:
The consumers of this API (particularly the agent community) do not want a separate API.This is why it's been combined. In the case of using a null pointer rather than an empty string, I don't understand how to implent this.

Reply:
Fixed this, need to test this works with wrappers, but given we're doing it in other modules I assume it should be fine.